tests/pr_test.go

```diff
@@ -110,9 +110,20 @@ func setupOptions(t *testing.T, prefix string, dir string, terraformVars map[str
 	os.Setenv("TF_VAR_sls_license_id", *slsLicenseId)
 
 	// Deploy Pre-requisite resources
-
 	realTerraformDir := "./resources"
 	tempTerraformDir, _ := files.CopyTerraformFolderToTemp(realTerraformDir, options.Prefix)
+	// Need to replace the symlinked file with the actual file since symlink will break when tests moves files to temp directory
+	override_location := tempTerraformDir + "/override.json"
+	e := os.Remove(override_location)
+	if e != nil {
+		t.Error(e)
+		return
+	}
+	err = files.CopyFile("../override-json-file/override.json", override_location)
+	if err != nil {
+		t.Error(err)
```

Comment:
A cleaner way to handle these errors, by failing the test and exiting, is to use the testify `require`, which checks the condition and fails tests and exits if not met. If you look down at line 131 you will start to see examples of this.